internal/cmd/gocdk/init_test.go

```diff
@@ -147,6 +148,83 @@ func TestInit(t *testing.T) {
 		}
 	})
 }
+func TestInferModulePath(t *testing.T) {
+	t.Run("NoGOPATHEntry", func(t *testing.T) {
+		dir, err := ioutil.TempDir("", testTempDirPrefix)
+		if err != nil {
+			t.Fatal(err)
+		}
+		defer func() {
+			if err := os.RemoveAll(dir); err != nil {
+				t.Error(err)
+			}
+		}()
+		ctx := context.Background()
+		pctx := &processContext{
```

Comment:
You can use `newTestProcessContext` for this now. You can override the `env` field to be what you need after that.